Frends.Excel.Parse/Frends.Excel.Parse/Excel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.ComponentModel;
+using System.Text;
+using ExcelDataReader;
+using Frends.Excel.Parse.Definitions;
+
+namespace Frends.Excel.Parse;
+
+public static class Excel
+{
+    /// <summary>
+    /// Converts Excel file to data set. [Documentation](https://github.com/FrendsPlatform/Frends.Excel/tree/main/Frends.Excel.Parse)
+    /// </summary>
+    /// <param name="input">Input configuration</param>
+    /// <param name="options">Input options</param>
+    /// <param name="cancellationToken"></param>
```

Comment:
I don't think the cancellation token needs to be documented, since it is passed automatically and doesn't require any user input. This may cause confusion to users if they read the documentation and then they try to find out how they can give the cancellation token for the task.

Reply:
This is honestly just for consistence. Otherwise it is a warning for visual studio and otherwise I think that it is somewhat nice to have all params enumerated, without selecting certain ones for not documenting.